src/shared-components/Card/ListItems/ListItems.tsx

```diff
@@ -0,0 +1,47 @@
+import {
+  Box,
+  List,
+  ListItem,
+  ListItemSecondaryAction,
+  ListItemText,
+  Stack,
+  Typography,
+} from '@mui/material';
+import ChevronRightIcon from '@mui/icons-material/ChevronRight';
+import { colors } from 'src/styles/colors';
+import { CheckBox } from '@mui/icons-material';
+
+export const ListItems = ({
+  title,
+  description,
+}: {
+  title: string;
+  description: string;
+}) => {
+  return (
+    <List>
```

Comment:
Create one generic ListItems element that will be a flex list with gap between elements as specified in Figma (make sure to use spacing from theme). It should not include any list items inside of it at this point but it should accept children.

Reply:
I think ı fixed it.